DigitalLearningSolutions.Data.Tests/Services/LoginServiceTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Services
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Data.Models.User;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Data.Tests.Helpers;
+    using FakeItEasy;
+    using NUnit.Framework;
+
+    public class LoginServiceTests
+    {
+        private ICryptoService cryptoService;
+        private LoginService loginService;
+        private IUserService userService;
+
+        [SetUp]
+        public void Setup()
+        {
+            userService = A.Fake<IUserService>();
+            cryptoService = A.Fake<ICryptoService>();
+
+            loginService = new LoginService(userService, cryptoService);
+        }
+
+        [Test]
+        public void Return_no_delegate_users_when_delegate_stored_password_is_empty()
```

Comment:
I feel like we probably want a few more tests here. A full set would probably look something like that:
-- GetUsersByUsername
* Returns admin and delegate users by username (to make sure what's returned by user service is passed back)

-- VerifyUsers
* Returns verified admin user
* Does not return unverified admin user
* Returns verified delegates
* Filters out unverified delegates
* Does not return any delegates if all unverified

-- Get admin user associated with delegate
* Returns nothing if delegate emial is empty
* Return nothing if no admin associated with delegate account
* Returns admin account associated with delegate